rs/bitcoin/kyt/BUILD.bazel

```diff
@@ -62,20 +63,19 @@ rust_canister(
     ],
 )
 
-rust_ic_test(
+test_using_pocket_ic_server(
     name = "integration_tests",
     srcs = ["tests/tests.rs"],
     data = [":btc_kyt_canister"],
     env = {
         "CARGO_MANIFEST_DIR": "rs/bitcoin/kyt",
         "IC_BTC_KYT_CANISTER_WASM_PATH": "$(rootpath :btc_kyt_canister)",
-        "POCKET_IC_BIN": "$(rootpath //rs/pocket_ic_server:pocket-ic-server)",
     },
+    macro = rust_ic_test,
     deps = [
         # Keep sorted.
         ":btc_kyt_lib",
         "//packages/pocket-ic",
-        "//rs/pocket_ic_server:pocket-ic-server",
         "//rs/test_utilities/load_wasm",
         "@crate_index//:candid",
     ],
```

Comment:
@dfinity/cross-chain-team is it important the bitcoin kyt tests are also run using the HEAD version of the pocket-ic server? Maybe on some schedule? If so I should set `extra_HEAD_tags = [ "some-tag" ],` here.

Reply:
I think it is okay to run with master or mainnet version of pocket-ic server.

Reply:
I do have a question though.. in case of a pocket-ic-server (or its related libraries) update, who will be notified of the possible breakage for these tests, and how are they notified?

Reply:
Since the Bitcoin KYT tests now use the mainnet pocket-ic server the tests will possibly break after mainnet is updated in the PR that updates the corresponding [MAINNET_REVISIO(https://github.com/dfinity/ic/pull/1468/files/e0ddc4d46ffea1245e5d834b01dcb664b2140299#diff-13270af6472df7268d3e162a8462748061a92ddbfd29f8f156bcb52813149646R14).

If we decide to run the Bitcoin KYT tests using the HEAD version of the pocket-ic server on a schedule (say nightly) we'll know of possible breakage as soon as the following night. Note that schedule-daily workflow failures are [reported](https://github.com/dfinity/ic/blob/master/.github/workflows/slack-workflow-run.yml) to slack.